__fixtures__/currentUserProfile.fixture.js

```diff
@@ -1,33 +1,32 @@
-function generateCurrentUserProfile (user) {
```

Comment:
@durdo should we change the fixtures style? Those are copied from Spotify response itself so I just wonder whether it is better to keep them untouched for testing purpose

Reply:
In call we decided that applying Prettier for the whole code would be better